src/main/java/me/karakelley/http/Request.java

```diff
@@ -0,0 +1,62 @@
+package me.karakelley.http;
+
+import java.io.BufferedReader;
+import java.util.HashMap;
+
+public class Request {
+  private final RequestLine requestLine;
+  private final RequestHeaders requestHeaders;
+  private final BufferedReader reader;
+
+  public Request(BufferedReader reader) {
+    this.reader = reader;
+    this.requestLine = setRequestLine();
+    this.requestHeaders = setHeaders();
+  }
+
+  public String getMethod() {
+    return requestLine.getMethod();
+  }
+
+  public String getPath() {
+    return requestLine.getPath();
+  }
+
+  public String getProtocol() {
+    return requestLine.getProtocol();
+  }
+
+  public HashMap getHeaders() {
```

Comment:
1. Why commit to a concrete implementation here?
2. Any specific reason for not specifying the types of the keys/values?